### PR TITLE
Hook Front button with back tutorial

### PR DESCRIPTION
**Pull request by RobertMili**
Here did i hook hint button with backend so i get map ID (because map Id is connected with tutorial).
Hook -> question button -> player.ts -> mapID -> tutorial ID.

### CODE REVIEW

File: src/components/game/MissionContainer.vue [hunk 1]
```diff
@@ -20,21 +67,10 @@ const showJson = () => {
...
-          You need to cross the bridge on the way to town. To move you here use the command
-          hero.moveTo(), where X is the marker (number) of the place you want to mov.
-          For Example, to move to location 1 on the map, type the command hero.moveTo(1)
-          You need to cross the bridge on the way to town. To move you here use the command
-          hero.moveTo(), where X is the marker (number) of the place you want to mov.
-          For Example, to move to location 1 on the map, type the command hero.moveTo(1)
-          You need to cross the bridge on the way to town. To move you here use the command
-          hero.moveTo(), where X is the marker (number) of the place you want to mov.
-          For Example, to move to location 1 on the map, type the command hero.moveTo(1)
+          {{ currentMap.tutorialdescription }}
```
Comment (JuliaCorell) [inline src/components/game/MissionContainer.vue]:
```suggestion
          {{ currentMap.tutorialhint }}
```

Comment (jLereback) [inline src/components/game/MissionContainer.vue:70]:
```suggestion
          {{ currentMap.tutorialhint }}
```

Comment (jLereback) [inline src/components/game/MissionContainer.vue:70]:
```suggestion
          {{ currentMap.tutorialhint }}
```

**Review by JuliaCorell — DISMISSED** [review]

File: src/components/game/MissionContainer.vue (hunk 1 above)
Comment (RobertMili) [reply src/components/game/MissionContainer.vue]: 👍

**Review by figgge — APPROVED** [review]
Great job!

**Pull request merged**